parquet/src/test/java/org/apache/iceberg/parquet/TestParquetMetrics.java

```diff
@@ -57,10 +78,133 @@ public File writeRecords(Schema schema, Map<String, String> properties, GenericD
     return ParquetWritingTestUtils.writeRecords(temp, schema, properties, records);
   }
 
-  @Override
-  public int splitCount(File parquetFile) throws IOException {
+  private int splitCount(File parquetFile) throws IOException {
     try (ParquetFileReader reader = ParquetFileReader.open(ParquetIO.file(localInput(parquetFile)))) {
       return reader.getRowGroups().size();
     }
   }
+
+  @Test
+  public void testMetricsForTopLevelWithMultipleRowGroup() throws Exception {
```

Comment:
I think we should keep these tests in the base file and ignore them for ORC using `Assume.assumeTrue(fileFormat() == FileFormat.PARQUET)`.

Reply:
What's the rationale for that? I think this test is specific to Parquet hence it does not need to be in the abstract `TestMetrics` since it used Parquet specifics like `PARQUET_ROW_GROUP_SIZE_BYTES`.

Reply:
If it depends on constants that can't easily be translated, then that's okay. The rationale is just to keep all the tests that we may eventually want to run on other formats in the same test file. This is minor, though.

Reply:
I added an `Assume` for this in #1070 and it defaults to not running the test.